allensdk/brain_observatory/behavior/behavior_project_cache/external/behavior_project_metadata_writer.py

```diff
@@ -0,0 +1,215 @@
+        ophys_experiments = self._behavior_project_cache.get_experiment_table(
+            suppress=suppress, as_df=False)
+        ophys_experiments = self._get_release_table(table=ophys_experiments)
+        self._write_metadata_table(df=ophys_experiments,
+                                   filename=output_filename)
+
+    def _get_release_table(self,
+                           table: Union[
+                               SessionsTable,
+                               BehaviorOphysSessionsTable,
+                               ExperimentsTable]) -> pd.DataFrame:
+        """Takes as input an entire project-level table and filters it to
+        include records which we are releasing data for
+
+        Parameters
+        ----------
+        table
+            The project table to filter
+
+        Returns
+        --------
+        The filtered dataframe
+        """
+        if isinstance(table, SessionsTable):
+            release_table = table.table.merge(self._release_behavior_only_nwb,
+                                              left_index=True,
+                                              right_index=True,
+                                              how='left')
+        elif isinstance(table, BehaviorOphysSessionsTable):
+            release_table = table.table
+        elif isinstance(table, ExperimentsTable):
+            release_table = table.table.merge(
+                self._release_behavior_with_ophys_nwb
+                    .drop('behavior_session_id', axis=1),
+                left_index=True,
+                right_index=True,
+                how='left')
+        else:
+            raise ValueError(f'Bad table {type(table)}')
+
+        return release_table
+
+    def _write_metadata_table(self, df: pd.DataFrame, filename: str):
+        """
+        Writes file to csv
+
+        Parameters
+        ----------
+        df
+            The dataframe to write
+        filename
+            Filename to save as
+        """
+        filepath = os.path.join(self._out_dir, filename)
+        self._logger.info(f'Writing {filepath}')
+
+        df = df.reset_index()
+        df.to_csv(filepath, index=False)
+
+        self._logger.info('Writing successful')
+
+    def _write_manifest(self):
+        data_files = \
+            self._release_behavior_only_nwb['isilon_filepath'].to_list() + \
+            self._release_behavior_with_ophys_nwb['isilon_filepath'].to_list()
+        filenames = OUTPUT_METADATA_FILENAMES.values()
+
+        def get_abs_path(filename):
+            return os.path.abspath(os.path.join(self._out_dir, filename))
+
+        metadata_files = [get_abs_path(f) for f in filenames]
+        data_pipeline = [{
+            'name': 'AllenSDK',
+            'version': allensdk.__version__,
+            'comment': 'AllenSDK version used to produce data NWB and '
+                       'metadata CSV files for this release'
+        }]
+
+        manifest = {
+            'data_files': data_files,
```

Comment:
Sorry to keep changing things up on you. Due to a recent comment from Scott, we decided that the `data_files` field is superfluous (file id and data file paths are already specified in metadata *.csv files) for the json file produced by this module.

Additional context:
https://github.com/AllenInstitute/informatics_data_release_tools/pull/17#discussion_r597199888